infrastructure/monitoring/promtail/README.md

```diff
@@ -0,0 +1,49 @@
+# Promtail
+
+Promtail is an agent which ships the contents of local logs to a private Grafana Loki instance. It is usually deployed to every machine that has applications needed to be monitored.
+It primarily:
+ - Discovers targets
+ - Attaches labels to log streams
+ - Pushes them to the Loki instance
+
+ ## Prerequisites
+
+Helm must be installed to use the charts value. Please follow the [Helm installing guide](https://helm.sh/docs/intro/install/) before getting starting.
+
+After Helm is ready, add the Grafana repo as follow:
+
+```helm
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+```
+To see the Grafana charts, you can run:
+
+```helm
+helm search repo grafana
+```
+
+## Installing
+
+To install the chart with the release name promtail and apply the configuration of values.yaml:
+
+```helm
+helm upgrade --install promtail -f < YOUR_PATH >/values.yaml grafana/promtail
+```
+
```

Comment:
Would add a bit of introduction, saying that next you are presenting the most relevant modifications in the promtail configuration.